.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "airbnb",
+
+    "rules": {
+        "indent": [2, 2],
```

Comment:
Can we not do 4-space throughout? It seems like a point of sensible consistency that all our code in all languages should be 4-space. Personally I find 2-space hard to read.